Sources/Utility/PkgConfig.swift

```diff
@@ -110,11 +138,15 @@ public struct PkgConfig {
     static func locatePCFile(
         name: String,
         customSearchPaths: [AbsolutePath],
-        fileSystem: FileSystem
+        fileSystem: FileSystem,
+        diagnostics: DiagnosticsEngine
     ) throws -> AbsolutePath {
         // FIXME: We should consider building a registry for all items in the
         // search paths, which is likely to be substantially more efficient if
         // we end up searching for a reasonably sized number of packages.
+        if pkgConfigSearchPaths == [] {
+            diagnostics.emit(PkgConfigError.pkgConfigExecutionError)
```

Comment:
Lets keep it a warning for now.

Reply:
Would that be emitting a `DiagnosticData` instead of an `Error` ?

Reply:
Yep, you can create a struct similar to [this](https://github.com/apple/swift-package-manager/blob/master/Sources/Commands/SwiftPackageTool.swift#L21).

Reply:
Thanks for the pointer! 🙇